### PR TITLE
Preserve date filter on registration table

### PR DESCRIPTION
**Pull request by flutistar**
*Issue #:* /bcgov/entity#23445
*Description of changes:*

- [ ] Passed defaultStartDate and defaultEndDate to DatePicker
- [ ] Updated placeholder text accordingly
- [ ] A function that returns PST date object.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-2047-vo6ktze4.web.app

File: ppr-ui/src/components/common/RangeDatePicker.vue [hunk 1]
```diff
@@ -30,6 +31,7 @@
         </b>
         <BaseDatePicker
           class="date-selection__picker mt-2"
+          :default-selected-date="endDate"
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/RangeDatePicker.vue:34]: Small thing but in this project we've been going with camelCase for the prop names

**Review by cameron-eyds — APPROVED** [review]
Looks good!
So the issue was just the display of the date in the picker? 
The date was still filtering by the date under the hood?

**Review by doug-lovett — APPROVED** [review]
LGTM

**Comment by flutistar** [comment]
> Looks good! So the issue was just the display of the date in the picker? The date was still filtering by the date under the hood?

Displaying the date and place holder text.
When the page is loaded, useRegistration sets placeholder text as empty.
And on the component, there is a `watch` function that initializing `startDate` and `endDate` if `dateTxt` is empty.
So we have both dates in store but not in local state.

I think that's why.

**Comment by flutistar** [comment]
@cameron-eyds 
FYI, I addressed your comment :)

**Pull request merged**